doc/Language/5to6-nutshell.pod6

```diff
@@ -12,7 +12,7 @@ features and idioms are not).
 Hence this should not be mistaken for a beginner tutorial or a promotional
 overview of Perl 6; it is intended as a technical reference for Perl 6
 learners with a strong Perl 5 background and for anyone porting Perl 5 code
-to Perl 6 (though note that L<#Automated translation> might be more
+to Perl 6 (though note that L<#Automated_translation> might be more
```

Comment:
Won't this underscore show up in the text?

Reply:
Is having `L<#Automated translation>` the same as `L<Automated translation|#Automated_translation>`?  

If that's the case, I'd say yes but I'm not sure though.